examples/gno.land/p/demo/grc/grc20/banker.gno

```diff
@@ -55,7 +58,11 @@ func (b *Banker) Mint(address std.Address, amount uint64) error {
 		return ErrInvalidAddress
 	}
 
-	// TODO: check for overflow
+	// check for overflow
+	if uint64_max-b.totalSupply < amount {
```

Comment:
Can you clean this up and make it something like `if diff := uint64_max-b.totalSupply; diff < amount {` and then use diff below instead of doing the same calculation again?

Reply:
commit 0356eca4